Libraries/CommonFunctions.psm1

```diff
@@ -2413,4 +2413,28 @@ Function ExtractSSHPublicKeyFromPPKFile {
         } else {
             return $null
         }
+}
+
+function Is-XDPCompatible() {
```

Comment:
Why this is a common function? it is common for XDP but not for others.

Reply:
It is a common function for XDP test-cases and not for others. I saw that this check will be repeating in all the test-cases so I wanted to increase re-usability of the code so I put it in CommonFunctions.psm1.

Reply:
@abhimarathe Can it be done in a separate PS script module for XDP? We have done a similar way in the bash script. You have dpdkUtils.sh for DPDK. If you see other test cases, there are a few having PS scripts in setupscript tag. You could have a separate PS module file and import in the run-time.

Reply:
Yes Ju,
I discussed this with Simon yesterday. A Separate module for XDP is not necessary as only one function will be there in that module. If we create one for Networking area, we can introduce all XDP, DPDK, SRIOV functions in that. What are your views on this?

Reply:
I removed this function and kernel version dependent check,